cmd/minishift/cmd/start.go

```diff
@@ -108,7 +109,12 @@ For the latter see 'minishift config -h'.`,
 		Run: runStart,
 	}
 
+	// enable experimental flags
+	//startCmd.Flags().BoolVar(&future, "future", false, "Enable experimental features (unsupported)")
+	future = os.Getenv("MINISHIFT_FUTURE") != ""
```

Comment:
Really ??? :-o

I think communication about this would happen on the PR and Issue that refer to these commands. In which topic would this else have a place?